index.html

```diff
@@ -111,7 +111,7 @@
    <li><a href=wd.html#keyboard>Keyboard</a>
    <li><a href=wd.html#mouse>Mouse</a>
   </ul></li>
- <li><a href=remote.html#remote_webdriver>Remote WebDriver</a>
+ <li><a href=remote.html#remote_webdriver>RemoteWebDriver</a>
```

Comment:
This does not describe the `RemoteWebDriver` interfaces, but the concept of a _remote WebDriver_.